### ISSUE TITLE
Lock dependencies

### ISSUE THREAD

**Issue by egorsmkv** [proger/uk#2]
It would be nice to have the list of dependencies in this project in a Poetry's `pyproject.toml` file

**Comment by proger** [comment]
Yeah I'm currently experimenting with mambaforge environments in another repo of ours. Condaforge has a kaldi build so that might kill two birds with one stone.
